### DISCUSSION TITLE
150% DPI显示有点问题

### DISCUSSION THREAD

**Discussion by my36z** [tamlok/vnote#2231] [General]
现在新的笔记本默认都是150%的缩放，这个时候vnote的缩放很奇怪，好像比其他软件的都大一点。

看这个 http://t.zoukankan.com/swarmbees-p-12004594.html
感觉好像是qt缩放到了200%

帮忙修正一下150%呗，毕竟现在笔记本都是这个配置

**Comment by tamlok** [comment]
是的，qt好像只支持整数倍缩放。

你可以强制100%缩放，或者直接缩放阅读或者编辑界面（鼠标滚轮加ctrl）

**Reply by my36z** [reply]
主要不是内容问题，是菜单和导航非常大，导致整个屏幕都被浪费了。
我试了一下vnote2，2版的菜单大小就很正常。只是界面上的个别字体有点小，不过整体感觉还行。

是否可以加一个参数强制dpi100%显示。
win10自带的调整效果很差，显示起来很模糊。

**Reply by tamlok** [reply]
windows里面程序右键兼容性那里可以选择dpi的参数吧？

菜单和导航栏可以通过vnotex.json里面修改大小，右上角修改用户配置文件，然后搜一下icon_size

**Reply by my36z** [reply]
windows里面程序右键兼容性那里被微软改了，之前的禁用缩放被删掉了。

改配置有个问题是接显示器的时候dpi变了，大小又变奇怪了。因为是笔记本，所以经常移动，如果是台式机也没那么多麻烦事情了。

**Reply by tamlok** [reply]
那你直接改配置文件修改图标大小吧
